commons-jcs-core/src/main/java/org/apache/commons/jcs3/auxiliary/disk/indexed/IndexedDiskCache.java

```diff
@@ -1259,7 +1259,7 @@ private long defragFile(final IndexedDiskElementDescriptor[] defragList, final l
     private IndexedDiskElementDescriptor[] createPositionSortedDescriptorList()
     {
         final List<IndexedDiskElementDescriptor> defragList = new ArrayList<>(keyHash.values());
-        Collections.sort(defragList, (ded1, ded2) -> Long.compare(ded1.pos, ded2.pos));
+        Collections.sort(defragList, Comparator.comparingLong(ded -> ded.pos));
```

Comment:
Don't you think the previous version is easier to understand?

Reply:
Hi @tvand 
I don't know why but i prefer using comparator. Of course it's a personal choice. I rollback this change 
TY